### ISSUE TITLE
Potential trouble shooting content?

### ISSUE THREAD

**Issue by Preeticp** [fabric8io/fabric8-online-docs#355]
hi guys

I'm trying the getting started guide

and got stuck at the step where it pushes the commit to github

it doesn't have my SSH key

I tried to give it using the preferences, but apparently that wasn't enough

also, the nice github button for using OAuth to create a new key and upload it didn't work

is that normal ?

eivantsov profile image	
eivantsov
3:27 PM
are you talking about Che IDE?

fromage profile image	
fromage
3:28 PM
yes

eivantsov profile image	
eivantsov
3:28 PM
and you created and started your ws from OSIO?

fromage profile image	
fromage
3:28 PM
yes

followed the getting started guide

from https://docs.openshift.io/getting-started-guide.html

got stuck at https://docs.openshift.io/getting-started-guide.html#committing_pushing_changes_git-hello-world

eivantsov profile image	
eivantsov
3:30 PM
let me check

New Messages
fromage profile image	
fromage
3:31 PM
thanks

eivantsov profile image	
eivantsov
3:38 PM
ok, so. everything is good with my account. here's what you might need to do.

In OSIO disconnect your GitHub account
In GitHub revoke access to OSIO
Logout, login again - you will be (should be) asked to connect your GitHub account. Do so.
You know what to do next.
Image Pasted at 2018-5-9 13-07.png

fromage profile image	
fromage
3:41 PM
ok trying

still can't push from che

This may be because you have not setup SSH keys or oAuth.
Check the documentation for using Git for more information on setting these up.

[also, real problem that the error messages can't be copied, had to use dev tools for that]

fromage profile image	
fromage
3:44 PM
2
am I supposed to see anything in Profile>Preferences SSH/VCS for Che?

eivantsov profile image	
eivantsov
3:45 PM
no

fromage profile image	
fromage
3:45 PM
1
I did keep my workspace, should I have created a new one?

eivantsov profile image	
eivantsov
3:45 PM
2
Commented on fromage's message: am I supposed to see anything in Profile>Preferences SSH/VCS for Che?
GitHub token should have been added to your account when you authorized the app

eivantsov profile image	
eivantsov
3:46 PM
1
Commented on fromage's message: I did keep my workspace, should I have created a new one?
you may want to try stopping this one and restarting or creating a new one

fromage profile image	
fromage
3:46 PM
ok

fromage profile image	
fromage
3:52 PM
ok that worked, I created a new workspace

how can I delete the other one?

I didn't find how: I can only stop it apparently

eivantsov profile image	
eivantsov
3:55 PM
not from OSIO.. but if you go to https://rhche.openshift.io/dashboard/#/workspaces you can do it from there

**Comment by mishaone** [comment]
@Preeticp can you summarize what you know about the requirements for this issue that we also support? If we don't know this, maybe we need to tag someone to clarify what we need to document for this issue.

**Comment by Preeticp** [comment]
@mishaone here's the summary:
Normally, GitHub token is added to your account when you authorize the app. However in this case, user's ssh key was unknown to Che work space initiated by OSIO. Hence he was unable to push from the Che work space.
Error seen: (Not sure if these are the exact words) Can't push from che
This may be because you have not setup SSH keys or oAuth.
Check the documentation for using Git for more information on setting these up.

Workaround: 
In OSIO disconnect your GitHub account
In GitHub revoke access to OSIO
Logout, login again - you will be (should be) asked to connect your GitHub account. Do so.
Also, you may want to try stopping the current workspace and restart or create a new one.

BTW our docs do not cover 'Deleting an exisiting workspace' in Che, but it seems like a good to know for users: We can not delete them from OSIO. but if you go to https://rhche.openshift.io/dashboard/#/workspaces you can do it from there.

**Comment by Preeticp** [comment]
This was a one time issue reported by one user do we want to document it?
Ooh, deleting existing workspace seems to be useful information imo, we should at least add a line with the link. Wdyt?

**Comment by rkratky** [comment]
@Preeticp Sounds like a good tidbit for the troubleshooting section.
